core/header.cpp

```diff
@@ -465,6 +465,12 @@ namespace MR
       WARN ("transform matrix contains invalid entries - resetting to sane defaults");
       transform() = Transform::get_default (*this);
     }
+    else {
```

Comment:
This bit is not strictly related to this, so might be best not to include... We'd had discussions in the past as to how best to handle non-orthogonal transforms - I guess this might be one way...

Reply:
I seem to recall Dave having commented proudly at some point about being able to apply affine transformations directly to the image header and being able to operate on those images / even open them in `mrview` without necessitating image regridding. So I don't think this is something that should be "slipped in" without more careful consideration.